.php_cs

```diff
@@ -1,33 +1,43 @@
 <?php
 
+$header = <<<'EOF'
+@copyright Copyright (C) eZ Systems AS. All rights reserved.
+@license For full copyright and license information view LICENSE file distributed with this source code.
+EOF;
+
 return PhpCsFixer\Config::create()
     ->setRules([
         '@Symfony' => true,
         '@Symfony:risky' => true,
         'concat_space' => ['spacing' => 'one'],
-        'array_syntax' => false,
+        'array_syntax' => ['syntax' => 'short'],
         'simplified_null_return' => false,
         'phpdoc_align' => false,
-        'phpdoc_separation' => false,
         'phpdoc_to_comment' => false,
         'cast_spaces' => false,
         'blank_line_after_opening_tag' => false,
         'single_blank_line_before_namespace' => false,
-        'phpdoc_annotation_without_dot' => false,
```

Comment:
I think that one is now not consisted across our repositories, why this have changed?